db-commons/src/main/scala/org/bitcoins/db/DbCommonsColumnMappers.scala

```diff
@@ -172,6 +172,11 @@ abstract class DbCommonsColumnMappers {
     MappedColumnType
       .base[SatoshisPerByte, Long](_.toLong, SatoshisPerByte.fromLong)
   }
+
+  implicit val addressTagMapper: BaseColumnType[AddressTag] = {
+    MappedColumnType
+      .base[AddressTag, String](_.name, tag => UnknownAddressTag(tag, ""))
```

Comment:
So everything from the database is of type `UnknownAddressTag`? this is not ideal :$

Reply:
At the end of the day, I think we can only provide a `addressTagMapper` of type `BaseColumnType[InternalAddressTag]`. We are going to have to require users using the external tag type to provide their own mapper that implements 

```scala
implicit val externalTagMapper: BaseColumnType[MyExternalTag] = ...
```

Reply:
If we do that, we then need to go back to the DAO for every type. We need to have a mapper that works for every kind of address tag.

Reply:
why wouldn't we do `InternalAddressTag.fromString(tag).getOrElse(UnknownAddressTag(tag)`?

Reply:
We can't do a `InternalAddressTag.fromString(tag)` without knowing the tag type.